### PR TITLE
need to sort batchlist by reference issue resolved

### LINKED ISSUE #996: Need to sort `BatchList` by `Reference Number`
We can't currently sort `BatchList` by `Reference` (it's not clickable).  We should be able to.

### PR DESCRIPTION
**Pull request by TahaKhanAbdalli**
fixes #996

### CODE REVIEW

File: src/resources/batches/BatchList.tsx [hunk 1]
```diff
@@ -180,6 +180,7 @@ export default function BatchList(): React.ReactElement {
         <FunctionField<Batch>
           label='Reference'
           render={(record) => `${record.vault?.[0]}${record?.batchNumber}`}
+          sortBy='id'
```
Comment (IanMayo) [inline src/resources/batches/BatchList.tsx:183]: I'm pretty sure the above should be sorted by `batchNumber`

**Review by IanMayo — CHANGES_REQUESTED** [review]
The data doesn't look like it is being sorted by batch reference.

File: src/resources/batches/BatchList.tsx (hunk 1 above)
Comment (TahaKhanAbdalli) [reply src/resources/batches/BatchList.tsx:183]: If it is okay for. Can I fix it tomorrow?

Comment (IanMayo) [reply src/resources/batches/BatchList.tsx:183]: Completely fine :-D.

Comment (TahaKhanAbdalli) [reply src/resources/batches/BatchList.tsx:183]: Thank you. @IanMayo

**Review by IanMayo — APPROVED** [review]

**Pull request merged**